assets/js/alma-widgets-inject.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Inject payment plan widget.
+ *
+ * @package Alma_WooCommerce_Gateway
+ */
+
+(function () {
+	var paymentPlanContainerId = "#alma-payment-plan";
+	var paymentPlanContainer   = jQuery( paymentPlanContainerId );
+	var jqueryUpdateEvent      = paymentPlanContainer.data( "jquery-update-event" );
+	var firstRender            = paymentPlanContainer.data( "first-render" );
+
+	window.AlmaInitWidget = function () {
+		paymentPlanContainer    = jQuery( paymentPlanContainerId ); // re-query element.
+		var enabledPlans        = paymentPlanContainer.data( "enabled-plans" );
+		var merchantId          = paymentPlanContainer.data( "merchant-id" );
+		var apiMode             = paymentPlanContainer.data( "api-mode" );
+		var amount              = parseFloat( paymentPlanContainer.data( "amount" ) );
+		var minAmount           = parseFloat( paymentPlanContainer.data( "min-amount" ) );
+		var maxAmount           = parseFloat( paymentPlanContainer.data( "max-amount" ) );
+		var amountQuerySelector = paymentPlanContainer.data( "amount-query-selector" );
+
+		if (amountQuerySelector) {
+			var amountElement = document.querySelector( amountQuerySelector );
+			if (amountElement) {
+				var child = amountElement.firstChild;
+				while (child) {
+					if (child.nodeType == 3) {
```

Comment:
Un commentaire pour expliciter ce que c'est qu'un `nodeType === 3` ?

```suggestion
		if (amountQuerySelector) {
			var amountElement = document.querySelector( amountQuerySelector );
			if (amountElement) {
				var child = amountElement.firstChild;
				while (child) {
					if (child.nodeType === 3) {
```

Reply:
j'ai mis `child.nodeType === Node.TEXT_NODE` pour être explicite ;)